metadata.rb

```diff
@@ -5,5 +5,5 @@
 description      'Provides an LWRP that facilitates access to the mixlibrary-core gem windows feature support.'
 long_description IO.read(File.join(File.dirname(__FILE__), "README.md"))
 
-version          '1.0.0' #Automatically versioned by CI system.  This version will always be out of date.
+version          '1.0.1' #Automatically versioned by CI system.  This version will always be out of date.

```

Comment:
So apparently we have shipped 1.0.22 already.

So we need to update Buildjob to do the same thing as our other repos to read version from metadata.

See:
http://supermarket.epnet.com/cookbooks/windows_feature
https://supermarket.chef.io/cookbooks/windows_feature